### PR TITLE
coveo-arnparse

### PR DESCRIPTION
**Pull request by jonapich**
This module deals with ARNs in python code.

### CODE REVIEW

File: coveo-arnparse/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# coveo-arnparse
+
+Simple dataclass around Amazon Resource NAmes (ARNs).
```
Comment (dotboris) [inline coveo-arnparse/README.md:3]:
```suggestion
Simple dataclass and parser around Amazon Resource Names (ARNs).
```

File: coveo-arnparse/coveo_arnparse/__init__.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    (?P<region>.*?):
+    (?P<account>.*?):
+    (?P<resource>.*?)
+    $
+    """,
+    re.VERBOSE,
+)
+
+
+@dataclass()
```
Comment (dotboris) [inline coveo-arnparse/coveo_arnparse/__init__.py:27]:
```suggestion
@dataclass
```

File: .github/workflows/coveo-arnparse.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        with:
+          python-version: 3.9
+
+      - name: Publish to pypi
+        uses: ./.github/workflows/actions/publish-to-pypi
+        with:
+          project-name: ${{ github.workflow }}
+          pypi-token: ${{ secrets.PYPI_TOKEN_COVEO_ARNPARSE }}
+          pre-release: ${{ github.ref != 'refs/heads/main' }}
+          dry-run: true
```
Comment (dotboris) [inline .github/workflows/coveo-arnparse.yml:62]:
```suggestion
          dry-run: ${{ github.ref != 'refs/heads/main' && github.event.inputs.publish != 'true' }}
```

All other projects use this in their workflow. Seems like it'd help when it comes time to release

**Review by dotboris — COMMENTED** [review]
Mostly small stuff

File: .github/workflows/coveo-arnparse.yml (hunk 3 above)
Comment (jonapich) [reply .github/workflows/coveo-arnparse.yml:62]: 🤦🏻‍♂️

**Review by dblanchette — APPROVED** [review]
Nice lib!

**Pull request merged**